src/components/ChatItem.tsx

```diff
@@ -47,7 +48,7 @@ type Props = {
  * Defaults to showing just the user-supplied message, if it has not yet been answered.
  * Otherwise, shows both question and answer, plus applicable references.
  */
-export const ChatItem = ({ question, answer, searchResults, onRetry, isStreaming }: Props) => {
+export const ChatItem = ({ question, answer, searchResults, factualConsistencyScore, onRetry, isStreaming }: Props) => {
```

Comment:
What's the reason for adding this new prop? It seems to me that at this point, we know we want to show the badge upon enabling FCS. My suggestion would be to integrate the badge into chat item, and instead expose a `showFactualConsistencyScore` prop, which toggles whether to show the badge or not. 

If it becomes necessary (e.g. we decide to expose an FCS display prop at the top level, which we currently don't), we can always add an optional render prop here.

This way, we also avoid conflating the idea of FCS being enabled and _how_ it's being displayed, which IMO we're kind of doing [here](https://github.com/vectara/react-chatbot/pull/42/files#diff-62b6fc952f585d3f67cdc6c883331ff7fa5caa6a627c8bef95b2a0fd544b560aR151).